specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+          "description": "Error code",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Description of the error",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "If applicable, would be used to indicate the property causing the error",
+          "type": "string",
+          "readOnly": true
+        },
+        "innerErrors": {
+          "description": "If applicable, inner errors would be returned for more details on the error",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "SendReadReceiptRequest": {
+      "description": "Request payload for sending a read receipt.",
+      "required": [
+        "chatMessageId"
+      ],
+      "type": "object",
+      "properties": {
+        "chatMessageId": {
+          "description": "Id of the latest chat message read by the user.",
+          "type": "string",
+          "example": "1592435762364"
+        }
+      }
+    },
+    "ChatMessagePriority": {
+      "description": "The chat message priority.",
+      "enum": [
+        "Normal",
+        "High"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ChatMessagePriority",
+        "modelAsString": true
+      }
+    },
+    "SendChatMessageRequest": {
+      "description": "Details of the message to send.",
+      "required": [
+        "content"
+      ],
+      "type": "object",
+      "properties": {
+        "priority": {
+          "$ref": "#/definitions/ChatMessagePriority"
+        },
+        "content": {
+          "description": "Chat message content.",
+          "type": "string",
+          "example": "Come one guys, lets go for lunch together."
+        },
+        "senderDisplayName": {
+          "description": "The display name of the chat message sender. This property is used to populate sender name for push notifications.",
+          "type": "string",
+          "example": "Bob Admin"
+        }
+      }
+    },
+    "SendChatMessageResult": {
+      "description": "Result of the send message operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "A server-generated message id.",
+          "type": "string",
+          "readOnly": true,
+          "example": "123456789"
+        }
+      }
+    },
+    "ChatMessage": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The id of the chat message. This id is server generated.",
+          "type": "string",
+          "readOnly": true,
+          "example": "123456789"
+        },
+        "type": {
+          "description": "Type of the chat message.\r\n            \r\nPossible values:\r\n    - Text\r\n    - ThreadActivity/TopicUpdate\r\n    - ThreadActivity/AddMember\r\n    - ThreadActivity/DeleteMember",
```

Comment:
We should probably remove "\r" and "\n"s from here.

Reply:
It is an unordered list that looks like this:
```
Type of the chat message.

Possible values:
- Text
- ThreadActivity/TopicUpdate
- ThreadActivity/AddMember
- ThreadActivity/DeleteMember
```

Wouldn't removing the new lines, make it hard to grok?

Reply:
Maybe it should become an extensible enum?